submissions/krylenger/html-css-popup/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>pop-up</title>
+    <link href="style.css" type="text/css" rel="stylesheet" />
+</head>
+
+<body>
+    <nav class="navigation-container">
+        <div class="navigation-list-container">
+            <ul class="navigation-list">
+                <li><a href="https://www.gmail.com/">Mail</a></li>
+                <li><a href="https://www.google.com/photos/about/">Images</a></li>
+                <li class="navigation-sub-menu">
+                    <input class='checkbox toggle-pop-up' type="checkbox" id="toggle-pop-up" />
+                    <label for="toggle-pop-up" class="toggle-pop-up-label navigation-list-icons"></label>
+                    <div class="pop-up-container">
+                        <ul class="pop-up-icons-container">
+                            <li class="item" tabindex="0"><img src="./icons/002-kitty-46.png"
+                                    alt="cat" /><span>Write</span></li>
+                            <li class="item" tabindex="0"><img src="./icons/016-kitty-33.png"
+                                    alt="cat" /><span>Work</span></li>
```

Comment:
Why in all item of the list you write `tabindex="0"`?

Reply:
Because li-item content contains noninteractive elements.
Wrap all li-item content by link and you don't need `tabindex` attribute.

Reply:
Didn't notice the lack of links 😞

Reply:
✅ changed to <a> elements